tsconfig.json

```
@@ -4,10 +4,10 @@
     "target": "es5",
     "lib": ["es5", "es6", "es7", "es2017", "dom"],
     "jsx": "react",
-    "rootDirs": ["src", "stories"],
+    "rootDirs": ["src", "src/stories"],
```

```suggestion
    "rootDirs": ["src"], 
```

should be sufficient. before the folders were on the same level, now stories are a subpath of the already included directory